### PR TITLE
chore: add dockerfile, add rollup id to cmd options

### PR DESCRIPTION
**Pull request by gupadhyaya**
## Summary by CodeRabbit

- **New Features**
	- Introduced a new command-line flag `rollup-id` for enhanced configuration.
	- Added support for rollup ID validation in transaction submissions.

- **Bug Fixes**
	- Improved error handling for invalid rollup IDs in transaction processing.

- **Tests**
	- Updated test cases to ensure correct handling of rollup IDs and error conditions.
	- Enhanced error handling checks in tests for mismatched rollup IDs.

### CODE REVIEW

**Review by Manav-Aggarwal — DISMISSED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Build stage
+FROM --platform=$BUILDPLATFORM golang:1.23-alpine AS build-env
+
+# Set environment variables
+ENV GO111MODULE=on
```
Comment (tzdybal) [inline Dockerfile:5]: Is it still needed? :thinking:

Comment (tzdybal) [inline Dockerfile:5]: Is it still needed? :thinking:

**Review by tzdybal — DISMISSED** [review]
LGTM

File: Dockerfile (hunk 1 above)
Comment (gupadhyaya) [reply Dockerfile]: nope. removed.

**Review by Manav-Aggarwal — APPROVED** [review]

**Review by tzdybal — APPROVED** [review]

**Pull request merged**